radio/src/opentx.cpp

```diff
@@ -1943,3 +1943,42 @@ uint32_t availableMemory()
   return ((uint32_t)((unsigned char *)&_heap_end - heap)) + info.fordblks;
 #endif
 }
+
+// Radio menu tab state
+#if defined(COLORLCD)
+bool radioThemesEnabled() {
+  return (g_model.radioThemesDisabled == OVERRIDE_GLOBAL && g_eeGeneral.radioThemesDisabled == 0) || (g_model.radioThemesDisabled == 2);
+}
+#endif
+bool radioGFEnabled() {
+  return (g_model.radioGFDisabled == OVERRIDE_GLOBAL && g_eeGeneral.radioGFDisabled == 0) || (g_model.radioGFDisabled == 2);
+}
+bool radioTrainerEnabled() {
+  return (g_model.radioTrainerDisabled == OVERRIDE_GLOBAL && g_eeGeneral.radioTrainerDisabled == 0) || (g_model.radioTrainerDisabled == 2);
+}
+
+// Mode menu tab state
```

Comment:
If you do many any changes, 'Model' typo ;)

Reply:
Fixed.